app/javascript/packages/document-capture/components/in-person-prepare-step.tsx

```diff
@@ -16,16 +17,34 @@ import { useI18n } from '@18f/identity-react-i18n';
 import { FormStepsButton } from '@18f/identity-form-steps';
 import UploadContext from '../context/upload';
 import MarketingSiteContext from '../context/marketing-site';
+import AnalyticsContext from '../context/analytics';
 import BackButton from './back-button';
 import InPersonTroubleshootingOptions from './in-person-troubleshooting-options';
 
 function InPersonPrepareStep({ toPreviousStep, value }) {
   const { t } = useI18n();
   const { inPersonURL } = useContext(FlowContext);
   const { flowPath } = useContext(UploadContext);
+  const { trackEvent } = useContext(AnalyticsContext);
   const { securityAndPrivacyHowItWorksURL } = useContext(MarketingSiteContext);
   const { selectedLocationName } = value;
 
+  const onContinue: MouseEventHandler = async (event) => {
+    removeUnloadProtection();
+
+    let destination;
+    if (event.target instanceof HTMLAnchorElement) {
+      event.preventDefault();
+      destination = event.target.href;
+    }
+
+    await trackEvent('IdV: prepare submitted');
```

Comment:
The idea with `await`-ing here is to allow the logging network request to finish before redirecting to the first Rails page of the flow. I had considered some alternatives like inferring a submission from a server-side request, but since the only route we have is a (repeatable/refresh-able) `GET` route, I couldn't determine a reliable way to infer it.

Reply:
do we have like a max wait timeout, or some sort of exception handling if the server is down for a bit?

Reply:
> do we have like a max wait timeout, or some sort of exception handling if the server is down for a bit?

Hm, there are ways to [add a timeout behavior to a fetch with `AbortSignal`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal#aborting_a_fetch_operation_with_a_timeout). Not sure if it makes sense to bake that into `trackEvent`'s default behavior?

`fetch` wouldn't normally throw an error if the request to the server was unsuccessful, so I'd expect that to be handled gracefully, but there are some cases like ["network error"](https://fetch.spec.whatwg.org/#dom-global-fetch) which could throw. Maybe `trackEvent` could silently absorb those, or we could add a `try / catch` or `try / finally` here ?